python/rpilocator-scraper/main.py

```diff
@@ -0,0 +1,40 @@
+import asyncio
+import feedparser
+from scramjet.streams import Stream
+
+
+# it will send data to the 'telegram-inbound' Topic
+provides = {
+    'provides': 'telegram-inbound',
+    'contentType': 'text/plain'
+}
+
+# specify models:
+MODELS = "CM4,PI4,PIZERO"
+
+# and regions:
+REGIONS = "UK,DE,PL"
+
+# and delay, min. 60 [seconds]
+DELAY = 60
+
+URL = f"https://rpilocator.com/feed/?country={REGIONS}&cat={MODELS}"
+
+async def scrape_rp(stream):
+    first_check = feedparser.parse(URL).entries
+    stock_ids=[entry.id for entry in first_check]
+
+    await asyncio.sleep(DELAY)
+
+    while True:
+        new_data = feedparser.parse(URL)
+        for entry in new_data.entries:
+            if entry.guid not in stock_ids:
```

Comment:
when not in stock or in stock?

Reply:
ok, I think I know what was meant. Not in stock, but shouldn't we write to the stream the entries which were found initially?

Reply:
Initially, I didn't want to write `first_data ` entries to the stream but I believe you are right. Changed it just now.

Reply:
thanks :)